### ISSUE TITLE
Resolve security vulnerabilities

### ISSUE THREAD

**Issue by dazza-codes** [sul-dlss/ld4p-marc21-to-xml#39]
#37 and #38 surface security vulnerabilities; find a way to resolve these, either using the features to skip some vulnerabilities that are commonly identified for backend databases (which apply to the database and not to the client code accessing the database) or update the dependencies to fix vulnerabilities that do apply to the code libraries.

**Comment by jgreben** [comment]
Closed by(?) https://github.com/sul-dlss/ld4p-marc21-to-xml/pull/41

I had to do something to resolve the dependency vulnerabilities to fix #29 but this needs review to see if it sufficiently fixes this issue.

**Comment by dazza-codes** [comment]
Agreed - if the updates in #41 are now passing the security vulnerability checks introduced in #38, this issue is resolved.  However, it's strange that the validate task reports there are no dependencies that it can check, see https://travis-ci.org/sul-dlss/ld4p-marc21-to-xml/builds/250855211#L2677-L2679 where it reports:
```
[INFO] --- dependency-check-maven:1.4.5:check (default) @ xform-marc21-to-xml ---

[INFO] No dependencies were identified that could be analyzed by dependency-check

[INFO] ------------------------------------------------------------------------
```

I don't have time to look into this, but I don't feel comfortable closing this issue until we understand this and ensure that the security check is working properly.

**Comment by jgreben** [comment]
See https://github.com/sul-dlss/ld4p-marc21-to-xml/pull/42

**Comment by dazza-codes** [comment]
Should be fixed by #42
